### DISCUSSION TITLE
CAN U HELP ME......

### DISCUSSION THREAD

**Discussion by DASHVANTH-03** [TimefoldAI/timefold-solver#228] [Ideas]
Hey!!
I'm a python beginner......
**Can u help me to write a program to find group of an atom in periodic table.**
THANK YOU ; D

**Comment by triceo** [comment]
I believe you have the wrong project for that.
I suggest you [ask your question on StackOverflow](https://stackoverflow.com/), and do a better job explaining your problem.
